src/plugins/intel_gpu/src/graph/activation.cpp

```diff
@@ -30,7 +30,8 @@ layout activation_inst::calc_output_layout(activation_node const& node) {
         activation_func::floor,
         activation_func::clamp };
 
-    if (input_node_layout.data_type == data_types::i8 || input_node_layout.data_type == data_types::i32) {
+    if (input_node_layout.data_type == data_types::i8 || input_node_layout.data_type == data_types::u8 ||
+        input_node_layout.data_type == data_types::i32) {
```

Comment:
Could you explain why this change is necessary?

Reply:
Kernel compile fails when activation=exp and data_type=u8. I thought restriction for i8 should be applied to u8 too.

Reply:
Then, what happens with this change? Does it fallback to different kernel?

Reply:
Discussed offline: 
This change is added to show better message in case of failure. Previously, error message was from kernel compilation. Now error message is from here. No behavioral change is expected.